renderer.js

```diff
@@ -99,6 +142,11 @@ class Song {
 }
 
 function playSong(e) {
+
+    if (e.target.classList.contains('btnDownload')) {
+        download(e);
```

Comment:
Because it is triggered on a "click" event on the row, and the row has `file_id` that we need to fetch in order to decide which song. 

(Unless I'm missing a simpler approach)